core/services/job/models.go

```diff
@@ -842,30 +843,44 @@ type LiquidityBalancerSpec struct {
 
 type WorkflowSpec struct {
 	ID            int32     `toml:"-"`
-	WorkflowID    string    `toml:"workflowId"`
+	WorkflowID    string    `toml:"workflowId"` // content hash
```

Comment:
@krehermann If this is the content hash, can we rename it as such? I expected ID to be the fully qualified ID, i.e. containing the owner and whatever else we need to be sure it's actually unique

Also, should we be automatically generating the content hash from the spec?

Reply:
Discussed offline, we're going to remove this comment and follow up later